platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileUtils.java

```diff
@@ -44,6 +44,11 @@ public final class SignedStateFileUtils {
 
     public static final String HASH_INFO_FILE_NAME = "hashInfo.txt";
 
+    /**
+     * The name of the file that contains the human-readable address book in the saved state
+     */
+    public static final String STATE_ADDRESS_BOOK_FILE_NAME = "stateAddressBook.txt";
```

Comment:
I think it will be obvious that this is the address book that we were using while this state was being written.

```suggestion
    public static final String STATE_ADDRESS_BOOK_FILE_NAME = "addressBook.txt";
```

Reply:
through discussion on slack we agreed to change the name to `currentAddressBook.txt`